src/components/Kanban/kanban.css

```diff
@@ -0,0 +1,13 @@
+@media (min-width: 641px) {
```

Comment:
For consistency - can we stick to using tailwind?

Reply:
I'll try to reduce the CSS to a minimum, but:
I haven't found a tailwind breakpoint for the exact width you're using for Desktop and I hesitated to edit the tailwind config. If you believe this could be useful, I could go ahead and add a custom "desktop" breakpoint in the config.
The other part of the CSS is a needed "hack" to fix drag and drop just on Desktop sizes and I'm not sure if I can query the selector with tailwind. (the [data-rbd-draggable-id] selector)
I'll look into it to see when that's applied.

Reply:
The fix for drag and drop (where part will go past the edge) is coming with the Hacker News PR. You can prep for it by adding `width=<whatever>` (Ie `width="48rem"`) as a prop to the DWrapper component wrapping your Kanban in Astrostation.tsx

As for the Tailwind breakpoint, its 'sm'. (Yes, its 640px vs 641px).

Reply:
I see, that's very useful info, thanks.
I should be able to remove most CSS then.

The only remaining part would be [data-rbd-draggable-id] and it's mostly to prevent elements from jumping around when dragging them and I haven't found another fix for this (and I'm not sure if your hacker news PR would fix that too since it seems to be more of an issue caused by nested draggables, see the currently known issues)

Reply:
I have no internal drags for the Hacker News PR, no nothing toughing that.